distributions/snowflake/build.yaml

```diff
@@ -0,0 +1,9 @@
+name: snowflake
+distribution_spec:
+  description: Use Snowflake for running LLM inference
+  providers:
+    inference: remote::snowflake
+    memory: meta-reference
+    safety: meta-reference
```

Comment:
Are you able to offer llama guard inference as well instead of relying on meta-reference?

Reply:
@raghotham This is actually desirable. Safety impl should be `meta-reference` -- the important question is that there be a `Llama-Guard` model available for inference.

Reply:
Unfortunately we don't have a llama-guard model hosted in our LLM service (see here for available models right now: https://docs.snowflake.com/user-guide/snowflake-cortex/llm-functions#availability). Could we use a vanilla llama instruct model for it instead?

I can mention llama-guard models to our llm service team, just not sure how it would align with roadmap / how quickly they'll be able to deploy.

Reply:
@ashwinb @raghotham please let me know if you have specific guidance / comments on updating the implementation given the above context, or if we are good to merge for now. In the future i'd like to build out additional components like memory...etc which Snowflake can support as well.